IdentityServer/Quickstart/TestUsers.cs

```diff
@@ -39,7 +39,7 @@ public static class TestUsers
         /// </summary>
         /// <param name="isProduction">if set to <c>true</c> [is production].</param>
```

Comment:
The param is currently removed and should be removed in the comment too.

Reply:
Thanks, I missed that!